src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getDescription() {
+        return "[" + this.getStatusIcon() + "] " + this.description;
+    }
+
+    public boolean checkIsDone() {
```

Comment:
Suggestion : Maybe you could adapt the method name because boolean methods should be named to sound like booleans.